docs/flank-output-investigation/flank_output_propositon.md

```diff
@@ -0,0 +1,101 @@
+# Flank output proposition
+
+## Android
+
+### Version section
+
+```bash
+
+version: local_snapshot
+revision: d5e7c76ab206373c6cdf0c1a25e2575f323e5929
+session id: 7a7ceda1-658e-461f-bd37-04f2b2a90a37
+
+```
+
+### Args section
+
+```bash
+<Diff of arguments>
+
+```
+
+### Dump shards
+
+```bash
+[Dump shards]
+  Saved 1 shards to /Users/adamfilipowicz/Repos/flank/results/2021-03-01_13-01-56.722225_lYrQ/android_shards.json
+  Uploading [android_shards.json] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+```
+
+### Uploading files
+
+```
+[Uploading files]
+
+  Uploading [app-debug.apk] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+  Uploading [app-single-success-debug-androidTest.apk] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+  Uploading [session_id.txt] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+```
+
+### Matrix info
+
+```
+[Matrix info]
+	Found 1 test / 1 shard
+
+  1 matrix ids created in 0m 5s
+  
+   matrix-1kozhsv2imkru https://console.firebase.google.com/project/flank-open-source/testlab/histories/bh.da0c237aaa33732/matrices/8233077922466140188/executions/bs.d3f60304f671ce86
+```
+
+
+
+### Storage information
+
+```bash
+
+  Raw results will be stored in your GCS bucket at [https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ]
+
+```
+
+### Test status
+
+```bash
+[Execution statuses]
+  3m 11s Test executions status: FINISHED:1
+  3m 11s matrix-1kozhsv2imkru FINISHED
```

Comment:
This could get long if there are a lot of matrices?

Reply:
it should not be so long, but it will be longer